src/components/ParsedBotMessageBody.tsx

```diff
@@ -81,7 +81,7 @@ export default function ParsedBotMessageBody(props: Props) {
             </BlockContainer>
           );
         })}
-        {sources.length > 0 ? (
+        {sources.length > 0 && enableSourceMessage ? (
```

Comment:
Just found that this condition needs to be here not on the above to make it work correctly.